lib/process-services-cloud/src/lib/form/models/form-cloud.model.ts

```diff
@@ -0,0 +1,247 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+    TabModel, FormWidgetModel, FormOutcomeModel, FormValues,
+    FormWidgetModelCache, FormFieldModel, ContainerModel, FormFieldTypes,
+    ValidateFormFieldEvent, FormFieldValidator, FormFieldTemplates } from '@alfresco/adf-core';
+import { FormCloudService } from '../services/form-cloud.service';
+
+export class FormCloudModel {
+
+    static UNSET_TASK_NAME: string = 'Nameless task';
```

Comment:
do we still need that?
in case yes, I'like to have a key that can be translated.

Reply:
Translated